editor/project_converter_3_to_4.cpp

```diff
@@ -3146,8 +3148,12 @@ void ProjectConverter3To4::process_gdscript_line(String &line, const RegExContai
 					line_new += starting_space + "# TODOConverter40 infinite_inertia were removed in Godot 4.0 - previous value `" + parts[5] + "`\n";
 				}
 
-				line_new += starting_space + base_obj + "move_and_slide()\n";
-				line = line_new + line.substr(0, start) + "velocity" + line.substr(end + start);
+				line_new += starting_space + base_obj + "move_and_slide()";
+
+				if (!line.begins_with(starting_space + "move_and_slide"))
+					line = line_new + "\n" + line.substr(0, start) + "velocity" + line.substr(end + start);
+				else
+					line = line_new + line.substr(end + start);
```

Comment:
You should use braces around statement blocks for `if` and `else`, even if it's a single line.

Reply:
Done